src/commander/commander.c

```diff
@@ -196,6 +197,17 @@ static commander_error_t _api_insert_remove_sdcard(const InsertRemoveSDCardReque
     return COMMANDER_OK;
 }
 
+static commander_error_t _api_get_root_fingerprint(RootFingerprintResponse* response)
+{
+    char fingerprint[4];
+    bool success = get_root_fingerprint(fingerprint, sizeof(fingerprint));
+    if (!success) {
+        return COMMANDER_ERR_INVALID_STATE;
```

Comment:
`COMMANDER_ERR_GENERIC` is better, as invalid state is used only by commander_states.c.